.gitignore

```diff
@@ -1,3 +1,7 @@
 .idea/
 target/
 *.iml
+/.settings/
```

Comment:
@mkordas I tried but 1) It seems to me it runs slower than Eclipse (my laptops' specs should be higher, I guess) and 2) I couldn't get used to the way it handles projects/workspace relationship; it's different than Eclipse ... but maybe that's just because I didn't play with it enough. Like I said, it runs annoyingly slowly.

Is there something teamed.io's projects that cannot or shouldn't be done in Eclipse?